src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -647,9 +647,11 @@ interfaces:
       - names[0]=Odyssey
       - shelves[0]=Classics
       output_specs:
+        - assign:
```

Comment:
I am thinking we should s/`assign`/`define`/g everywhere. It more accurately names what we're doing (variable definition) and it leaves the name `assign` free should we ever need to implement just a plain old assign.

Thoughts?

Reply:
Fine with me :)